src/pages/AppDashboard.vue

```diff
@@ -106,10 +107,28 @@ async function fetchAndPopulateUsersAndActions() {
     download: stats.data.actions?.download,
     upload: stats.data.actions?.upload,
     delete: stats.data.actions?.delete,
-    transfers: stats.data.actions?.transfers,
+    transfers: stats.data.actions?.ownership_change,
     share: stats.data.actions?.share,
     revoke: stats.data.actions?.revoke,
   }
+
+  const storage: number = stats.data.actions?.storage
```

Comment:
isn't the type be `number | undefined` ?

Reply:
Noted. Actually we'll need to remove `?` from actions. I kept it just in case if anything changes in backend and if for some reason actions doesn't come then our app will break.